apiary_v2.apib

```diff
@@ -1231,4 +1231,670 @@ the listed orders by their creation date.
           }
 
 
+# Group Factory API
```

Comment:
Do we want to call it Production Orders API instead of Factory API?

Reply:
Fair. Whatever makes more sense and we start to call it that way. I think you might have used the term "Production Order" somewhere else.

Reply:
`Manufacturing Order API` then?

Reply:
I think calling the whole thing the "Factory API" still makes sense since it's on `factory.unmade.com/v2`.